relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -736,6 +740,131 @@ fn extract_captured_substring<'a>(string: &'a str, pattern: &'a Lazy<Regex>) ->
     None
 }
 
+/// Returns the category of a span from its operation. The mapping is available in:
+/// <https://develop.sentry.dev/sdk/performance/span-operations/>
+fn span_op_to_category(op: &str) -> Option<&str> {
+    Some({
+        // General
+        if op.starts_with("mark") {
+            "mark"
+        } else if op.starts_with("function") {
+            "function"
+        }
+        //
+        // Browser
+        else if op.starts_with("pageload") {
+            "pageload"
+        } else if op.starts_with("navigation") {
+            "navigation"
+        } else if op.starts_with("resource") {
+            "resource"
+        } else if op.starts_with("browser") {
+            "browser"
+        } else if op.starts_with("measure") {
+            "measure"
+        } else if op.starts_with("ui") {
+            "ui"
+        } else if op.starts_with("http") {
+            "http"
+        } else if op.starts_with("serialize") {
+            "serialize"
+        }
+        //
+        // JS frameworks
+        else if op.starts_with("ui.react") {
+            "ui.react"
+        } else if op.starts_with("ui.vue") {
+            "ui.vue"
+        } else if op.starts_with("ui.svelte") {
+            "ui.svelte"
+        } else if op.starts_with("ui.angular") {
+            "ui.angular"
+        } else if op.starts_with("ui.ember") {
+            "ui.ember"
+        }
```

Comment:
This can never happen, since on the line `765` you will have a match with `ui` only.